arc/database/client.py

```
@@ -579,6 +579,22 @@ async def ensure_scenario_exists(self, scenario_id: str, scenario_data: Optional
                 })
 
     # Outcome Recording (Optimized for TimescaleDB hypertable)
+    def _augment_trajectory(self, trajectory: Dict[str, Any], outcome_data: Dict[str, Any]) -> Dict[str, Any]:
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)